### ISSUE TITLE
Moon in dark/light mode switch is a bit small 

### ISSUE THREAD

**Issue by D0J0P** [elementary/icons#1092]
### What Happened?

The moon in the dark/light mode switch is visually smaller than the sun(and the maximize button next to it too I guess). 

### Steps to Reproduce

![Screenshot from 2022-03-28 21 27 17](https://user-images.githubusercontent.com/19495660/160514446-f5084a60-e816-4c1f-a68e-1106a33109b4.png)

### Expected Behavior

That the moon would be about the same size as the sun icon.

### OS Version

6.x (Odin)

### Software Version

Latest release (I have run all updates)

**Comment by danrabbit** [comment]
Yeah in general the sizing of the weather icons could probably use another look
